spyder/plugins/history.py

```diff
@@ -230,6 +232,11 @@ def add_history(self, filename):
         editor.toggle_wrap_mode( self.get_option('wrap') )
 
         text, _ = encoding.read(filename)
+        linebreaks = [m.start() for m in re.finditer('\n', text)]
+        maxNline = CONF.get('historylog', 'max_entries')

```

Comment:
Plugins can access config their own options like this

    maxNline = self.get_option('max_entries')